README.md

```diff
@@ -7,48 +7,56 @@
 
 # Tracee - Container and system tracing using eBPF
 
-Tracee is a lightweight and easy to use container and system tracing tool. It allows you to observe system calls and other system events in real-time. A unique feature of Tracee is that it will only trace newly created processes and containers (that were started after Tracee has started), in order to help the user focus on relevant events instead of every single thing that happens on the system (which can be overwhelming). Adding new events to Tracee (especially system calls) is straightforward, and will usually require no more than adding few lines of code.
-
-Other than tracing, Tracee is also capable of capturing files written to disk or memory ("fileless"), and extracting binaries that are dynamically loaded to an application's memory (e.g. when an application uses a packer). With these features, it is possible to quickly gain insights about the running processes that previously required the use of dynamic analysis tools and special knowledge.
+Tracee is a lightweight and easy to use container and system tracing tool. It allows you to observe system calls and other system events in real-time. A unique feature of Tracee is it's comprehensive filtering abilities which allows you to focus on relevant events instead of every single thing that happens on the system. Unlike other tracing tools, Tracee is a security tool, which is demonstrated by features like capturing IoC artifacts from running applications, tracing non-syscall security events, and producing security insights.
```

Comment:
I wanted to remove this whole uniqueness part, but I remember you wanted to add it in the first place. are you ok with removing it? so it will be just what you wrote, and the rest they can read about later

Reply:
The situation is now different, because we no longer filter new processes by default, so modifications are needed, and I understand that.
If you are writing about the unique filters as I wrote - that should be enough